### ISSUE TITLE
Watchtower client vs server GUI

### ISSUE THREAD

**Issue by Jacksper13** [mynodebtc/mynode#519]
There is currently a single Watchtower switch, that's really the Watchtower server switch.

It would be great if there was a Watchtower client switch (enabled with `lnd --wtclient.active`) and then you could add one or more watchtower server uris (that would do `lncli wtclient add 032...63@1.2.3.4:9911`).

More info [here](https://github.com/lightningnetwork/lnd/blob/master/docs/watchtower.md)

Then other showing other watchtower client options in the GUI like showing the info of each connected tower or the stats (`lncli wtclient tower` or `lncli wtclient stats`) woudl be great. I wonder if Watchtower should have its own page or section within the Lightning wallet, there's a lot of potential in this option.

Thanks!

- [x] Mark this if you searched the existing list of open/closed issue and did not find anything similar

**Comment by Jacksper13** [comment]
I actually kinda wanna have a second pi with mynode community edition as a watchtower plugged in a different physical location looking over my main node. I feel like having a broad and working watchtower functionality would differentiate mynode form other similar plug and play solutions - AFAICT none of the other have any watchtower functionality

**Comment by tehelsper** [comment]
Yeah, it would be nice to have a separate watchtower page. However, the toggle should be enabling client and server.

**Comment by Jacksper13** [comment]
Interesting, I had to reboot mynode after hitting "Save" on watchtower (after hitting the switch) for it to load properly. After hitting save it looks like everything disappears in the GUI and feels like the channels closed (which panicked more than one in the TG group haha). Also, yeah, just checked and you are right in that it enables client and server, my bad.

A GUI to add towers and see stats would still be great though.

**Comment by tehelsper** [comment]
You shouldn't need to reboot - changing the config does need to restart lnd. I can probably display a better status while it's restarting.

**Comment by tehelsper** [comment]
I'll add the separate page to my todo list.

**Comment by Jacksper13** [comment]
I'll close when added =)
